
image_width: 640
image_height: 480
camera_name: markers_camera
camera_matrix:
  rows: 3
  cols: 3
  data:  [ 818.84969068912301, 0., 323.60431160857354, 0., 817.80600966254758, 249.52519704664763, 0., 0., 1. ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [ -0.11951213511222493, 1.3325193226947083, 0.0012794955574293634, -0.00081011144713774867, -4.7873354586767647 ]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 0.0]
